src/libraries/System.Private.CoreLib/src/System/Reflection/TypeAttributes.cs

```diff
@@ -23,6 +23,7 @@ public enum TypeAttributes
         AutoLayout = 0x00000000,     // Class fields are auto-laid out
```

Comment:
" 0 is AutoLayout, 0x2 is SequentialLayout, 4 is ExplicitLayout" comment above is bogus